docs/specs/code/SERVER_STATUS_ENDPOINT.md

```diff
@@ -0,0 +1,37 @@
+# Abstract
+
+Describe a new endpoint to notify server issues in Yoroi-frontend.
+   
+# Motivation
+
+Currently, Yoroi-frontend has no way to notify neither server issues nor connection issues. Also, once Yoroi has the ability to notify server issues, the endpoint used by Yoroi to fetch the information can be used to create a public URL that reports the mentioned status, such as [AWS](https://status.aws.amazon.com/) and [Twitter](https://status.twitterstat.us).
+
+# Proposal
+
+Adding a new endpoint in the backend named `api/status` is proposed. This endpoint has the method `GET` that returns the last JSON inserted in the following table.
+
+```
+CREATE TABLE api_statuses (
+   ID serial NOT NULL PRIMARY KEY,
+   status json NOT NULL
+);
+```
+
+The element insertion will be performed manually into the database.
```

Comment:
This is fine, but I would expect to explain the tradeoffs and why you are proposing this.